stripe-attestation/src/main/java/com/stripe/attestation/IntegrityStandardRequestManager.kt

```diff
@@ -68,9 +70,16 @@ class IntegrityStandardRequestManager(
     private suspend fun request(
         requestHash: String?,
     ): Result<String> = runCatching {
+        if (integrityTokenProvider == null) {
+            // If prepare() hasn't been called yet, perform it preemptively.
+            // This ensures callers of requestToken() don't need to explicitly prepare first.
+            // This is a convenience method, but it is recommended to call prepare() early as
+            // it can take a few seconds to complete.
+            prepare().getOrThrow()
+        }
```

Comment:
this covers potential (rare) cases where app recreation causes the singleton instance to be cleared.